doc/classes/Line2D.xml

```diff
@@ -210,44 +213,63 @@
 	</methods>
 	<members>
 		<member name="begin_cap_mode" type="int" setter="set_begin_cap_mode" getter="get_begin_cap_mode" enum="Line2D.LineCapMode">
+			The style used for the shape in the first point of the line.
```

Comment:
Don't use passive voice (used), try to use the direct tense instead, and use `'s` to mark possession:
`Controls the style of the line's first point.`